src/arrayMethodSort.js

```diff
@@ -5,7 +5,37 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let counter = 0;
+
+    function compareStrings(a, b) {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() === b.toString()) {
+        return 0;
+      } else {
+        return -1;
+      }
+    }
+
+    const callback = (compareFunction === undefined)
+      ? compareStrings
+      : compareFunction;
```

Comment:
You can replace this by passing compareStrings as a default argument for compareFunction
Like this:

`[].__proto__.sort2 = function(compareFunction = compareStrings)`